charts/xui-webapp/values.preview.template.yaml

```diff
@@ -22,6 +22,7 @@ nodejs:
      FEATURE_WORKALLOCATION_ENABLED: true
      SERVICES_CASE_JUDICIAL_API: http://rd-judicial-api-aat.service.core-compute-aat.internal
      SERVICES_LOCATION_REF_API_URL: http://rd-location-ref-api-aat.service.core-compute-aat.internal
+     SERVICES_TRANSLATION_API_URL: http://ts-translation-service-aat.service.core-compute-aat.internal
```

Comment:
Can you confirm if you have added this variable to Custom-Environment-Variable.json file?

Reply:
@udaydenduluri33  Yes it is available